docs/development/style-guide.rst

```diff
@@ -0,0 +1,237 @@
+********************************************************
+Astropy Style Guide: A Writing Resource for Contributors
+********************************************************
+
+Abbreviations
+=============
+
+Place abbreviations such as i.e. and e.g. within parentheses, where they are
+followed by a comma. Alternatively, consider using "that is" and “for example”
+instead, preceded by an em dash or semicolon and followed by a comma, or
+contained within em dashes.
+
+Examples:
+---------
+* The only way to modify the data in a frame is by using the ``data`` attribute
+  directly and not the aliases for components on the frame (i.e., the following
+  will not work).
+* There are no plans to support more complex evolution (e.g., non-inertial
+  frames or more complex evolution), as that is out of scope for the ``astropy``
+  core.
+* Once you have a coordinate object you can access the components of that
+  coordinate — for example, RA or Dec — to get string representations of the
+  full coordinate.
+
+Capitalization
+==============
+
+Capitalize all proper nouns (names) in plain text, except when referring to
+package/code names, in which case use lowercase and double backticks.
+
+Examples:
+---------
+* Follow Astropy guidelines for contributing code.
```

Comment:
Astropy vs astropy -- I still find this confusing, because in the papers, we enforce that capitalized Astropy means the project, and lowercase astropy means the core package. Should this apply in the doc too and be explained here? I am actually not sure if we are consistent in the doc ourselves...

Reply:
Yes, there is a lot of inconsistency in the documentation between capitalized and lowercase Astropy, so this guideline is intended to help with that. I thought that the examples illustrated what you are saying here, Astropy meaning the project and ``astropy`` meaning the core package, but I'll add that to the guideline as well and let me know if you think that helps clear it up. Thanks!